hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NodeEndpoint.java

```diff
@@ -148,6 +148,7 @@ public Response getDatanodes() {
           .withBuildDate(nodeManager.getBuildDate(datanode))
           .withLayoutVersion(
               dnInfo.getLastKnownLayoutVersion().getMetadataLayoutVersion())
+          .withNetworkLocation(datanode.getNetworkLocation())
           .build());
```

Comment:
Thanks @symious.  Is it useful to define an interface to get the network location from the nodeManager as well, such as defining `nodeManager.getNetworkLocation` that returns `inMemoryDatanodeDetails` for the networkLocation?